# -*- coding: utf-8 -*-
# Generated by Django 1.10.4 on 2016-12-31 01:14
from __future__ import unicode_literals

import django.contrib.auth.validators
from django.db import migrations, models


class Migration(migrations.Migration):

    dependencies = [
        ('customers', '0003_auto_20161231_0020'),
    ]

    operations = [
        migrations.AddField(
            model_name='customer',
            name='confirmed_company_details',
            field=models.BooleanField(default=False),
        ),
        migrations.AddField(
            model_name='customer',
            name='confirmed_phone_number',
            field=models.BooleanField(default=False),
        ),
        migrations.AlterField(
            model_name='customer',
            name='username',
            field=models.CharField(error_messages={'unique': 'A user with that username already exists.'}, help_text='Required. 150 characters or fewer. Letters, digits and @/./+/-/_ only.', max_length=150, unique=True, validators=[django.contrib.auth.validators.UnicodeUsernameValidator()], verbose_name='username'),
        ),
    ]
